File: bsp4rs/src/com/jetbrains/bsp/bsp4rs/python/python_build_server.rs
use crate::*;

#[derive(Debug)]
pub enum BuildTargetPythonOptions {}

impl Request for BuildTargetPythonOptions {
    type Params = PythonOptionsParams;
    type Result = PythonOptionsResult;
    const METHOD: &'static str = "buildTarget/pythonOptions";
}

File: bsp4kt/src/org/jetbrains/bsp/bsp4kt/PythonBuildServer.kt
package org.jetbrains.bsp.bsp4kt

import org.jetbrains.jsonrpc4kt.services.JsonNotification
import org.jetbrains.jsonrpc4kt.services.JsonRequest
import kotlinx.serialization.SerialName
import java.util.concurrent.CompletableFuture

interface PythonBuildServer {
  @JsonRequest("buildTarget/pythonOptions")
  fun buildTargetPythonOptions(params: PythonOptionsParams): CompletableFuture<PythonOptionsResult>

}
